### PR TITLE
9 use case submitword

### PR DESCRIPTION
**Pull request by cotopaxi64**

### CODE REVIEW

**Review by mariana0412 — APPROVED** [review]
Looks good!

File: src/main/java/usecases/submit_word/SwInputData.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+ */
+public class SwInputData {
+    private String word;
+
+    private String playerId;
+
+    /**
+     * The constructor of SwInputData.
+     * @param word the word that has been submitted.
+     * @param playerId the Id of the player that attempted to submit the word.
```
Comment (joel-mat-03) [inline src/main/java/usecases/submit_word/SwInputData.java:15]: minor (unimportant) comment: for more readability, consider using tab to evenly space parameter name and description

**Review by joel-mat-03 — DISMISSED** [review]
Follows all the coding conventions.

**Review by AlekseyPanas — DISMISSED** [review]
All code looks awesome. The unit test could perhaps be shortened by bringing some classes outside of the methods and making them customizable for multiple tests, but I think we all made a mess with the unit tests this first time around. If it ain't broke, don't fix it. Good work!

**Review by bac423 — DISMISSED** [review]
Awesome. Everything seems in order. Documentation is very clear

**Review by AlekseyPanas — DISMISSED** [review]

**Review by mariana0412 — APPROVED** [review]
Good job!

**Review by Serdiuk-Andrii — APPROVED** [review]

**Review by mariana0412 — APPROVED** [review]

**Pull request merged**